core-sdk-samples/higgs-shop-sample-app/src/layouts/App/mpconfig.prod.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
Why is this file being committed within the layouts?

Reply:
accidental